src/Angor/Avalonia/AngorApp.Model/Domain/Projects/Project.cs

```diff
@@ -1,4 +1,4 @@
-namespace Angor.UI.Model.Implementation.Projects;
+namespace AngorApp.Model.Domain.Projects;
```

Comment:
I am not sure I like the name Domain in the ns, `AngorApp.Model.Domain.Projects` is descriptive enough no?